src/main/kotlin/stringAddCalculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,44 @@
+package stringAddCalculator
+
+object StringAddCalculator {
+    private const val CUSTOM_DELIMITER =
+        """//(.)\\n(.*)"""
+    private const val ZERO = 0
+    private const val NEGATIVE_NUMBER_THROW_MESSAGE = "can not use nagative number"
+    private val delimiters = mutableListOf(",", ":")
+
+    fun add(text: String): Int {
+        if (text.isBlank()) return 0
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        if (text.isBlank()) return ZERO
```